notifique/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,13 @@
       android:label="@string/label_application"
       android:supportsRtl="true"
       tools:ignore="GoogleAppIndexingWarning">
+    <service android:name=".NotificationListener"
+             android:label="@string/service_name"
```

Comment:
https://developer.android.com/guide/topics/manifest/service-element.html
Recommended here.

But, I know of no practical use for it. I'm going to say revert.